controllers/project_controller.go

```diff
@@ -137,6 +137,14 @@ func (r *ProjectReconciler) handleCreate(req ctrl.Request, instance *jiraservice
 	}
 
 	log.Info("Successfully created Jira Service Desk Project: " + instance.Spec.Name)
+
+	err = r.JiraServiceDeskClient.UpdateCustomerAccessStatus(project.Key)
+	if err != nil {
+		return reconcilerUtil.ManageError(r.Client, instance, err, false)
+	}
+
+	log.Info("Successfully updated the Service Desk Open Access Permissions to Added Customers Only")
```

Comment:
Successfully updated the Service Desk Open Access Permissions to Added Customers Only
```suggestion
	log.Info("Successfully updated the Access Permissions to customer")
```